src/client/editor/components/StandardMaterialComponent.js

```diff
@@ -62,6 +62,8 @@ export default class StandardMaterialComponent extends SaveableComponent {
       this._object[map] = texture;
     } catch (e) {
       // TODO Should show warning on texture property when this happens.
+      const resource = this._mapTypesToTextureType(map);
```

Comment:
I would just pass the original `propertyName` in from `updateProperty`, instead of doing a reverse lookup again.